src/DbSelect.php

```diff
@@ -82,7 +84,7 @@ public function __construct(
      * @param  int $itemCountPerPage Number of items per page
      * @return array
      */
-    public function getItems($offset, $itemCountPerPage)
+    public function getItems($offset, $itemCountPerPage): array
```

Comment:
Adding type declarations here is a problem, because this class is not `final`, so subtypes are potentially broken.

Reply:
Removed in new PR